video/out/vo_gpu_next.c

```diff
@@ -1883,6 +1883,9 @@ static void update_render_options(struct vo *vo)
     p->peak_detect.smoothing_period = opts->tone_map.decay_rate;
     p->peak_detect.scene_threshold_low = opts->tone_map.scene_threshold_low;
     p->peak_detect.scene_threshold_high = opts->tone_map.scene_threshold_high;
+#if PL_API_VER >= 263
```

Comment:
I don't think this is needed since we already require PL_API_VER >= 264 for libplacebo-next feature